<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuSE::JsArchitecturalMetricsConfig</class>
 <widget class="QDialog" name="DuSE::JsArchitecturalMetricsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>84</width>
     <height>33</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="scriptsMetricsTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>475</width>
     <height>181</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Script File</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="runPushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>84</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QComboBox" name="metricDefaultComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>The Architectural Metric Default:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>The Architectural Metrics Options:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setDefaultPushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>95</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Set as default</string>
   </property>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>scriptsMetricsTable</zorder>
  <zorder>runPushButton</zorder>
  <zorder>setDefaultPushButton</zorder>
  <zorder>metricDefaultComboBox</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DuSE::JsArchitecturalMetricsConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DuSE::JsArchitecturalMetricsConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
